log/stdlog/stdlog.go

```diff
@@ -59,17 +61,24 @@ type Logger struct {
 func (sl Logger) Named(name string) *Logger { //nolint:gocritic // we pass by value to get a copy
 	sl.name = name
 
-	if name != "" {
-		name = "[" + name + "] "
-	}
-
-	sl.errorPfx = name + "[ERROR] "
-	sl.infoPfx = name + "[INFO] "
-	sl.debugPfx = name + "[DEBUG] "
+	sl.errorPfx = logLinePrefix(sl.labels, name, "ERROR")
+	sl.infoPfx = logLinePrefix(sl.labels, name, "INFO")
+	sl.debugPfx = logLinePrefix(sl.labels, name, "DEBUG")
 
 	return &sl
 }
 
+func logLinePrefix(labels []string, name, level string) string {
```

Comment:
It should be also called in `New`, shouldn't it?

Reply:
Why?

Reply:
Logs won't have labels until `Named()` is called.

Reply:
Good point.